artemis/reporting/modules/ssh_bruter/translations/pl_PL/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,30 @@
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:2
+msgid ""
+"The following addresses contain SSH servers where simple login/password "
+"pair allows logging in:"
+msgstr ""
+"Na następujących adresach uruchomione są serwery SSH do których można się"
+" zalogować prostym hasłem:"
+
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:6
+msgid "the following credentials allow logging in"
+msgstr "Poniższe dane umożliwiają zalogowanie się"
+
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:10
+#, fuzzy
+msgid "username:"
+msgstr "nazwą użytkownika"
+
+#: artemis/reporting/modules/ssh_bruter/template_exposed_ssh_with_easy_password.jinja2:11
+#, fuzzy
+msgid "and "
+msgstr "i"
```

Comment:
I'm against fuzzy matches, and besides here you translate and with space to "i" without space - that's asking for trouble ;)

Reply:
polish version returns "ihasłem", so you need to preserve the space, preferably outside of translation